src/controllers/trust.details.controller.ts

```diff
@@ -0,0 +1,99 @@
+import { NextFunction, Request, Response } from 'express';
+import * as config from '../config';
+import { logger } from '../utils/logger';
+import { getApplicationData, setExtraData } from '../utils/application.data';
+import * as mapperToPageService from '../utils/trust/mapper.to.page';
+import * as mapperToSessionService from '../utils/trust/mapper.to.session';
+import { ApplicationData } from '../model/application.model';
+import * as PageModel from '../model/trust.page.model';
+import { Trust, TrustKey } from '../model/trust.model';
+
+const TRUST_DETAILS_TEXTS = {
+  title: 'Tell us about the trust',
+};
+
+const get = (
+  req: Request,
+  res: Response,
+  next: NextFunction,
+): void => {
+  try {
+    logger.debugRequest(req, `${req.method} ${req.route.path}`);
+
+    const appData: ApplicationData = getApplicationData(req.session);
+
+    const trustId = req.params['id'];
+    const pageData: PageModel.TrustDetails = mapperToPageService.mapDetail(
```

Comment:
This appears to be using explicit mapping, as opposed to the implicit mapping used elsewhere in the app.
My gut instinct is that I prefer explicit mapping (it breaks quickly and loudly when the mapping code ought to change, as opposed to sometimes failing silently)

However, I would much rather be consistent with the rest of the app, e.g.
https://github.com/companieshouse/overseas-entities-web/blob/main/src/controllers/beneficial.owner.individual.controller.ts#L55

Reply:
After talking with @markpit & @mwestacott, we're OK with this explicit mapping model as-is, provided that there is a comment in any explicit mappers noting that they behave differently to the mapping elsewhere in the app.